apps/docs/src/content/reference/extras/html.mdx

```diff
@@ -54,8 +71,75 @@ This component is a port of [drei's `<Html>` component](https://github.com/pmndr
 The container of your `<Canvas>` component needs to be set to `position: relative | absolute | sticky | fixed`. This is because the DOM element will be mounted as a sibling to the `<canvas>` element.
 </Tip>
 
+## Stopping and Starting the Task
+
+\<HTML> has an `autoRender` prop that you can use to turn off and on its render task. If at some point in your application, you no longer need to update the hmtl, you can set `autoRender` to `false`. If you need to resume the task, set `autoRender` back to `true`.
+
 <Example path="extras/html/basic" />
 
+\<HTML> also exports it's internal render task and the `startRendering`, `stopRendering`, and `render` functions so you can either render on demand or start and stop the internal task at your will.
+
+```svelte title="Render-On-Demand"
+<script>
+  let html = $state()
+
+  $effect(() => {
+    // if (shouldRender) {
+    html?.render()
+    // }
+  })
+</script>
+
+<HTML
+  autoRender={false}
+  bind:this={html}
+>
+  <h1>Hello World</h1>
+</HTML>
+```
+
+```svelte title="Start-And-Stop-Rendering"
+<script>
+  let html = $state()
+
+  // turn this on and off in accordance with your application
+  let renderWhileTrue = $state(false)
+
+  $effect(() => {
+    if (html !== undefined) {
+      if (renderWhileTrue) {
+        html.startRendering()
+        // always stop rendering if it was started
+        return () => {
+          html.stopRendering()
+        }
+      }
+    }
+  })
+</script>
+
+<HTML
+  autoRender={false}
+  bind:this={html}
+>
+  <h1>Hello World</h1>
+</HTML>
+```
+
+In both cases you should set `autonRender` to false so that the render task doesn't automatically begin.
+
+Lastly, you can access these functions from the `<HTML>`'s children snippet.
+
+```svelte
+<HTML autoRender={false}>
+  {#snippet children({ render, startRendering, stopRendering })}
```

Comment:
I'm still learning svelte 5 snippets... in the Component Signature we've put `start`, `stop`, `render`, I take it order and spelling aren't an issue here?

Reply:
oh yeah gotta update that. ✅ i went with `startRendering` over just `start` that way you know what you're starting. it would be more evident if the component were named `HTMLRenderer` but i think this is okay.